sailesh_work/terraform/backend.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {
+    bucket = "devopsremotestate1"
+    key    = "terraform.tfstate"
```

Comment:
pls create the folder environment specific so that other people can understand the environment specific tfstate file

Reply:
done